src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp

```diff
@@ -61,6 +61,16 @@ class SubgraphBaseTest : public CommonTestUtils::TestsCommon {
     virtual std::vector<ov::Tensor> get_plugin_outputs();
 };
 
+inline std::vector<InputShape> dynamic_shapes_to_test_representation(const std::vector<ov::PartialShape>& shapes) {
+    std::vector<InputShape> result;
+    for (const auto& staticShape : shapes) {
+        if (staticShape.is_dynamic())
+            throw std::runtime_error("dynamic_shapes_to_test_representation can process only static shapes");
```

Comment:
Not clear to understand why we can work with static shapes only in `dynamic_shapes_to_test_representation`. What should do this function? Please provide3 more applicable name..
And: 
`result.push_back({{staticShape}, {staticShape.get_shape()}});` ?? why do use such construction? if shapes are static? We can extend `static_shapes_to_test_representation` for the case

Reply:
Right, `static_shapes_to_test_representation` is used to transform `std::vector<ov::Shape>` -> `std::vector<InputShape>`. However, shapes are represented as `std::vector<ov::PartialShape>` in dynamic tests, so I thought it would be nice to have a similar function that transforms `std::vector<ov::PartialShape>` -> `std::vector<InputShape>`.

Reply:
No problem, I can add `static_shapes_to_test_representation` overload that takes `std::vector<ov::PartialShape>` as an argument.